common_design_subtheme/site.webmanifest

```
@@ -0,0 +1,20 @@
+{
+    "name": "Common Design Demo",
+    "short_name": "CD",
+    "icons": [
+        {
+            "src": "img/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "img/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#1f69b3",
+    "background_color": "#ffffff",
+    "display": "standalone",
+    "start_url": "."
```

```suggestion
    "start_url": "/"
```

When I look at [MDN docs for start_url](https://developer.mozilla.org/en-US/docs/Web/Manifest/start_url#relative_url) it says:

> If the URL is relative, the manifest URL is used as the base URL to resolve it.

So if I'm understanding correctly, a value of `.` would end up pointing to `/sites/themes/custom/common_design_subtheme/`